webhooks/virtualmachine/mutation/virtualmachine_mutator.go

```diff
@@ -751,11 +751,18 @@ func SetDefaultControllers(
 	if vm.Spec.Hardware == nil {
 		vm.Spec.Hardware = &vmopv1.VirtualMachineHardwareSpec{}
 	}
-	vm.Spec.Hardware.IDEControllers = []vmopv1.IDEControllerSpec{
-		{BusNumber: 0},
-		{BusNumber: 1},
+
+	if len(vm.Spec.Hardware.IDEControllers) == 0 {
```

Comment:
Maybe just nil check instead since a user trying to create w/ no IDE is also invalid